app/controllers/passthrough_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PassthroughController < ApplicationController
+  after_action :skip_authorization
+
+  def index
+    current_organisation = document_models.find { |model| model.organisations.include? current_user.organisation_content_id }
+
+    if current_user.gds_editor?
+      redirect_to "/aaib-reports"
+    elsif current_organisation
+      redirect_to("/" + current_organisation.slug) && return
```

Comment:
I had to use brackets after `redirect_to` so that using `&&` worked. Even though `and` would be correct, it seems like it is a [known problem](https://github.com/bbatsov/rubocop/issues/1487) with Rubocop (which govuk-lint is made from) so I don't think I could have changed anything in govuk-lint to make it work. Is a bit annoying but I think it is small enough to not be too much of an issue.